pages/guide/04-spa-deployments.md

```diff
@@ -1,14 +1,193 @@
 # 04 SPA deployments
 
-If you have built a Lustre application that doesn't have a backend you need to
-deploy, you can use a number of static hosting services to get your SPA online.
-In this guide we'll look at two options, GitHub Pages and Netlify.
+If you have built a Lustre application that doesn't have a backend you need to deploy, you can use a number of static hosting services to get your SPA online. In this guide we'll look at two options, GitHub Pages and Cloudflare Pages, but the principles can be applied to other services as well.
```

Comment:
both these options are written with automated deployments in mind and therefore assume that you have a GitHub project.

it should be possible to deploy to Cloudflare Pages without a GitHub project I think. not something that I have tested, and I am not sure how valuable it will be in this case.

for the full stack project I agree that it makes sense since it's a more complicated process in general